### PR TITLE
Bugfix/fix psql index status

### PR DESCRIPTION
**Pull request by Tschuppi81**
Search: Fix search cli for index status

TYPE: Bugfix
LINK: None

### CODE REVIEW

File: src/onegov/search/cli.py [hunk 1]
```diff
@@ -39,7 +53,9 @@ def psql_index_status(app: 'Framework') -> None:
             success = 3
             continue
 
-        q = session.query(func.count(model.fts_idx))
+        count_column = get_non_nullable_columns(model)[0]
+        q = session.query(func.count(count_column))
```
Comment (Daverball) [inline src/onegov/search/cli.py:57]: Have you tried:

```suggestion
        q = session.query(model).with_entities(func.count(text('*'))
```

That seems a lot more robust to me than inspecting the table, that should generate `SELECT count(*) FROM ...`

**Review by Daverball — COMMENTED** [review]
I'd try to get rid of the introspection overhead, since it doesn't seem necessary.

File: src/onegov/search/cli.py (hunk 1 above)
Comment (Tschuppi81) [reply src/onegov/search/cli.py:57]: Will do so

Comment (Tschuppi81) [reply src/onegov/search/cli.py]: Works as expected, just commited

**Review by Daverball — APPROVED** [review]

**Pull request merged**